--- cli/test/cli/help_test.dart
+++ cli/test/cli/help_test.dart
@@ -27,8 +27,9 @@
          -A - Runs 'git add -A' before other actions. Which adds tracked and untracked files in whole working tree.
          -a - Runs 'git add .' before other actions. Which adds tracked and untracked files in current folder and subfolders.
          -b, --rebase-prefer-base - Runs 'stax rebase --prefer-base' afterwards on all children branches.
+         -d, --rebase-dont-resolve - Runs 'stax rebase --dont-resolve' afterwards on all children branches.
          -m, --rebase-prefer-moving - Runs 'stax rebase --prefer-moving' afterwards on all children branches.
-         -r, --rebase - Runs 'stax rebase' afterwards on all children branches.
+         -r, --rebase - Runs 'stax rebase' afterwards on all children branches (default).
          -u - Runs 'git add -u' before other actions. Which adds only tracked files in whole working tree.
  • commit - Creates a branch, commits, and pushes it to remote. First argument is mandatory commit message. Second argument is optional branch name, if not provided branch name would be generated from commit message.
       Positional arguments:
@@ -75,7 +76,8 @@
          -a, --abandon - Abandon rebase that is in progress, stax can't abort own rebases.
          -b, --prefer-base - Prefer base changes on conflict.
          -c, --continue - Continue rebase that is in progress.
-         -m, --prefer-moving - Prefer moving changes on conflict.
+         -d, --dont-resolve - Do not automatically resolve conflicts.
+         -m, --prefer-moving - Prefer moving changes on conflict (default).
  • settings - View or modify stax settings
       Positional arguments:
          arg1 - Subcommand (add, clear, remove, set, show)
